spec/game_spec.rb

```diff
@@ -0,0 +1,17 @@
+require 'game'
+describe Game do
+#   describe '#score' do
+#     it 'calculates the score' do
+#       game = Game.new
+#       20.times {game.hits(2)}
+#       expect(game.score).to eq 40
+#     end
+#   end  
+# end
+it 'can do' do
```

Comment:
It would be good to add a bit more detail on what the test is for so it is easy to understand and separate it from other tests